doc/fileexplorer.rst

```diff
@@ -18,7 +16,7 @@ Performing file operations
 
 You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and (on Windows) opening an external file explorer.
```

Comment:
```suggestion
You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and opening an external file explorer.
```

Small change because in the screenshot people can see how to open files in Finder.

Reply:
Sure